composer.json

```diff
@@ -15,7 +15,10 @@
         "symfony-cmf/routing-auto": "^2.0",
         "symfony-cmf/routing-bundle": "^2.0",
         "aferrandini/urlizer": "1.0.*",
-        "phpdocumentor/reflection-docblock": "^3.1, !=3.2.0"
+        "phpdocumentor/reflection-docblock": "^3.1, !=3.2.0",
+        "doctrine/orm": "~2.5.0",
+        "symfony-cmf/core-bundle": "^2.0",
```

Comment:
why do you need the core-bundle?

Reply:
Because we use `Symfony\Cmf\Bundle\CoreBundle\Translatable\TranslatableInterface` to check if an entity is translatable